delineate.py

```diff
@@ -717,14 +734,15 @@ def save_pickle(geotype: str, gdf: gpd.GeoDataFrame, basin: int, high_resolution
             gdf.sindex.create_index()
 
         # Get the standard project filename for the pickle files.
-        pickle_fname = get_pickle_filename(geotype, basin, high_resolution)
+        pickle_fname = get_pickle_filename(conf, geotype, basin, high_resolution)
         if not os.path.isfile(pickle_fname):
-            if VERBOSE: print(f"Saving GeoDataFrame to pickle file: {pickle_fname}")
+            if conf.VERBOSE: 
+                print(f"Saving GeoDataFrame to pickle file: {pickle_fname}")
             try:
                 pickle.dump(gdf, open(pickle_fname, "wb"))
-            except:
-                raise Warning("Could not save pickle file to: {pickle_fname}")
+            except Exception as e:
+                raise Warning(f"Could not save pickle file to: {pickle_fname}. Message: {str(e)}")
```

Comment:
here I changed the exception handling a bit to log out a message of the actual error.